src/linkPanel/linkPanel.ts

```diff
@@ -0,0 +1,51 @@
+import { JSONObject } from '@lumino/coreutils';
+import { BoxPanel } from '@lumino/widgets';
+
+import { IGlueSessionSharedModel } from '../types';
+
+/**
+ * The link editor widget.
+ */
+export class LinkWidget extends BoxPanel {
+  constructor(options: LinkWidget.IOptions) {
+    super({ ...options, direction: 'left-to-right' });
+    this._sharedModel = options.sharedModel;
+    this.id = 'glue-link-editor';
```

Comment:
Do we need `id`?

Reply:
Right, it's copied from the previous PR when it was a main area widget.

Reply:
Fixed